components/formats-gpl/src/loci/formats/in/DeltavisionReader.java

```diff
@@ -161,6 +162,11 @@ public boolean isThisType(String name, boolean open) {
   /* @see loci.formats.IFormatReader#isThisType(RandomAccessInputStream) */
   @Override
   public boolean isThisType(RandomAccessInputStream stream) throws IOException {
+    // Extra sanity check; if it's a TIFF, return false immediately, to avoid
+    // false negatives.
+    if(new TiffParser(stream).isValidHeader()) {
+      return false;
+    }
```

Comment:
Unfortunately, this isn't the case.  The length check still passes with TIFF data, including the test data from 17411, so this is needed to be sure it's not TIFF.  The dv magic and length checks are simply too ambiguous on their own to guarantee a correct `false` return, unless there's something more definitive we can check for.

This check will at least cover most TIFF-based formats to prevent misbehaviour, though ideally it wouldn't be needed.